decision-process.md

```diff
@@ -12,15 +12,35 @@ reified and talked upon, rather than hidden behind individual initiatives.
 2. Members of the Github organization DlangGraphicsWG have no special rights, 
    apart from being able to merge/reject Pull Requests according to this document.
 
-3. Modifications proposals to the Output are brought in through Pull Requests.
+3. Modifications proposals to the Output (.md files) are brought in through Pull Requests.
 
-4. Voting is open to anyone, as long as the PR is currently open, for a period of at least 24h.
-   After this time, a member of the Github Organization is allowed to merge or reject the Pull Request as long as the PR is still open by its author
+4. Each open Pull Request is a vote to be held. 
```

Comment:
This doesn't have to be the case (corrections, additions etc.)

Reply:
You should make a pull request with that amendment so we can vote on it ;)